src/config/config.js

```diff
@@ -42,6 +42,9 @@ const config = {
         312: process.env.HUB_REPORT_URL_312,
         644: process.env.HUB_REPORT_URL_644,
     },
+    featureFlags: {
+        transferVerification: process.env.FEATURE_FLAG_TRANSFER_VERIFICATION !== 'false',
```

Comment:
```suggestion
        transferVerification: process.env.FEATURE_FLAG_TRANSFER_VERIFICATION.toLowerCase() !== 'false',
```

Reply:
mmm...im just following the established pattern above.

Reply:
What's the case we want to happen almost every time? Basically, make it always do that except for a perfect explicit match.

Reply:
there is a line above doing the same thing without .toLowerCase(). Are you saying it is in scope of this change to fix that as well?

Reply:
also breaks if there is not env var set with that name. Suggest using a module like "env-var" in future.